src/components/tickets/components/Tickets.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import TicketMenu from './Menu';
+import TicketTable from './Table';
+
+const Tickets = () => (
+    <div>
+        <TicketMenu />
+        <br />
+        <TicketTable />
```

Comment:
use css instead of a break here. Add a class to your `div` and use flex box

Reply:
No line breaks please